DiscordBotsList.Api.Adapter.Discord.Net/DiscordBotsList.Api.Adapter.Discord.Net.csproj

```diff
@@ -2,23 +2,23 @@
 
     <PropertyGroup>
         <TargetFramework>net8.0</TargetFramework>
-        <Authors>Velddev, Faith Viola</Authors>
+        <Authors>Velddev, Faith</Authors>
         <Company>Top.gg</Company>
         <Product>DiscordBotsList.Api.Adapter.Discord.Net</Product>
-        <Description>Adapter for Discord.Net</Description>
+        <Description>Top.gg API adapter for Discord.Net</Description>
         <PackageReleaseNotes>Initial release</PackageReleaseNotes>
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
         <Copyright>Mike Veldsink</Copyright>
-        <PackageTags>discord bots list org wrapper api discord.net</PackageTags>
+        <PackageTags>discord bots topgg api discord.net</PackageTags>
         <RepositoryType>git</RepositoryType>
-        <RepositoryUrl>https://github.com/DiscordBotList/DBL-dotnet-Library</RepositoryUrl>
-        <PackageProjectUrl>https://github.com/DiscordBotList/DBL-dotnet-Library</PackageProjectUrl>
-        <Version>1.5.0</Version>
+        <RepositoryUrl>https://github.com/Top-gg-Community/dotnet-sdk</RepositoryUrl>
+        <PackageProjectUrl>https://github.com/Top-gg-Community/dotnet-sdk</PackageProjectUrl>
+        <Version>2.0.0</Version>
```

Comment:
This does not warrant a breaking change. Please refer to https://semver.org

Reply:
Forgot to revert that (that was lifted from #29), sorry.